pkg/logger/file_hook.go

```diff
@@ -0,0 +1,54 @@
+// Copyright © 2022 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logger
+
+import (
+	"fmt"
+	"path/filepath"
+	"runtime"
+	"time"
+
+	rotatelogs "github.com/lestrrat-go/file-rotatelogs"
+	"github.com/rifflock/lfshook"
+	"github.com/sealerio/sealer/common"
+	"github.com/sirupsen/logrus"
+)
+
+func NewFileHook(filePath string) (logrus.Hook, error) {
+	if filePath == "" {
+		filePath = common.DefaultLogDir
+	}
+
+	path := filepath.Join(filePath, "sealer.log")
+	writer, _ := rotatelogs.New(
+		path+".%Y%m%d",
```

Comment:
Whether the file name should be suffixed with `.log` and we give the prefix `sealer` or a custom prefix name.

Reply:
sealer.log is the softlink file name. the specific log file name with the timedata as suffix.